README.md

```diff
@@ -18,11 +18,40 @@ mbed import mbed-os-example-bootloader
 cd mbed-os-example-bootloader
 ```
 
-## Set up application to be a bootloader
+## Connecting the SD card
+This example uses the **onboard SD card port** for the K64F and the UBLOX_EVK_ODIN_W2.
 
-All supported boards mentioned above are set up to build as a bootloader image. To add support for a new board, you must specify the size of the bootloader.
+For other targets, the following Arduino form-factor SPI pins are used by default:
```

Comment:
It's a difficult habit to stop. A trick I use is to Ctrl+F "is". If the word immediately after is a verb that ends with "ed", it's passive voice [examples: is used, is needed, is modified]. I do the same thing with "are" [examples: are used, are needed, are modified].